.isort.cfg

```diff
@@ -0,0 +1,3 @@
+[settings]
+
```

Comment:
Probably not? But I think it looks better to separate the header from the contents by a blank line.